src/auto-approve-and-land-prereleases.js

```diff
@@ -24,14 +37,17 @@ module.exports = robot => {
     const isMemberOfFusionOrg = await getIsMemberOfOrg(github, 'fusionjs', pr.user.login);
     if(!(isPrerelease && isMemberOfFusionOrg)) return;
 
+    // Wait for any quick-running statuses to complete
+    await delay(3 * 1000);
```

Comment:
Not elegant, but helps mitigate the race to mark build check as `success`.

Reply:
Hmm, this has a funny smell to it. What status are we waiting for? Just feels like we should be able to do something with the github hooks to avoid.

Reply:
Agreed.  This isn't great.  An ideal solution would be to ensure (1) all of the checks are super quick and (2) we wait for all to complete before landing.

In practice, some of our checks are slow (and fairly variable in terms of execution time).  Buildkite checks can take on the order of minutes to complete and are (in theory) unnecessary as pre-releases introduce no new code changes.

I'm not convinced we can achieve these goals without quite a bit of re-work, so this was the quick and dirty solution that's "good enough" (TM).  Delay lets the really quick ones run, and implicitly says "whatever isn't done in the timeframe we deem reasonable doesn't matter".

Alternatively, if we don't care much for quick checks, we can remove this and instead track when all statuses have succeeded and then attempt to land.  Pros: less smell; Cons: potentially much slower pre-releases.

TL;DR - `delay` ensures that the quick-running probots have time to add labels, flag malformed titles, etc. without worrying about the long-running jobs (like buildkite's).

Reply:
FWIW, I'd vote to push for faster pre-releases and revisit down the  road if we feel to need to.